presto-native-execution/presto_cpp/main/PrestoServer.h

```diff
@@ -170,7 +170,7 @@ class PrestoServer {
   virtual std::string getBaseSpillDirectory() const;
 
   /// Invoked to enable stats reporting and register counters.
-  virtual void enableRuntimeMetricReporting();
+  virtual void enableWorkerStatsReporting();
```

Comment:
A small question: I guess the purpose of changing this name(`"RuntimeMetric"` -->  `"WorkerStats"`) here is to distinguish between "metric" (process level stats) and task-reported metrics(at runtime), so as to reduce some confusion?   

However, in this PR, "PrometheusStatsReporter.h" is placed in the "runtime-metrics" directory, so,  the same confusion still exists.

Reply:
That is right. This requires multiple touch points to be modified (in docs and code). For now we decided stay with this naming. We will revisit this.

Reply:
We should name these TaskMetrics or WorkerMetrics. Both can live under runtime-metrics.